.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+before_install:
+  # For tests running hg command
+  # set up username
+  - echo '[ui]' > ~/.hgrc
+  - echo 'username = anybox-test' >> ~/.hgrc
+
+language: python
+
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - pip install .
+  - pip install coveralls
+  - pip install flake8
+  - pip install bzr
```

Comment:
doc seems to be obsolete. We can also put all these requirements in a requirements.txt file and call pip install -f requirements.txt